src/ray/protobuf/gcs.proto

```diff
@@ -463,10 +466,12 @@ message ResourcesData {
   // Indicates whether resource normal task is changed. Only used when
   // light heartbeat enabled.
   bool resources_normal_task_changed = 11;
+  // The timestamp that normal task resources are measured.
+  int64 resources_normal_task_timestamp = 12;
```

Comment:
Can we make this 13 to avoid changing the order?

Reply:
I just put all three `resources_normal_tasks_*` fields at the last part.